incubator/jaeger/templates/collector-deploy.yaml

```diff
@@ -25,10 +24,9 @@ spec:
 {{ toYaml .Values.collector.podAnnotations | indent 8 }}
 {{- end }}
       labels:
-        app: {{ template "jaeger.name" . }}
-        component: collector
-        release: {{ .Release.Name }}
-        jaeger-infra: collector-pod
+        app.kubernetes.io/name: {{ template "jaeger.name" . }}
```

Comment:
In all deployments/statefulsets/daemonsets changed, you should define a selector in order to avoid being blocked in each version: https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md#deployments-statefulsets-daemonsets-selectors

Reply:
Good call thanks! Changes applied, please review.